File: Medii de proiectare si programare/java/lab4/src/main/java/com/example/temampp/domains/Show.java
package com.example.temampp.domains;

import java.time.LocalDate;

public class Show extends Entity<String> {
    private String showName;
    private String artistName;
    private LocalDate date;
    private String placeOfShow;
    private Integer noTicketsAvailable;
    private Integer noTicketsSold;

    public Show(String showName, String artist, LocalDate date, String placeOfShow, Integer noTicketsAvailable, Integer noTicketsSold) {
        this.showName = showName;
        this.artistName = artist;
        this.date = date;
        this.placeOfShow = placeOfShow;
        this.noTicketsAvailable = noTicketsAvailable;
        this.noTicketsSold = noTicketsSold;
    }

    public String getShowName() {
        return showName;
    }

    public void setShowName(String showName) {
        this.showName = showName;
    }

    public String getArtistName() {
        return artistName;
    }

    public void setArtistName(String artistName) {
        this.artistName = artistName;
    }

    public LocalDate getDate() {
        return date;
    }

    public void setDate(LocalDate date) {
        this.date = date;
    }

    public String getPlaceOfShow() {
        return placeOfShow;
    }

    public void setPlaceOfShow(String placeOfShow) {
        this.placeOfShow = placeOfShow;
    }

    public Integer getNoTicketsAvailable() {
        return noTicketsAvailable;
    }

    public void setNoTicketsAvailable(Integer noTicketsAvailable) {
        this.noTicketsAvailable = noTicketsAvailable;
    }

    public Integer getNoTicketsSold() {
        return noTicketsSold;
    }

    public void setNoTicketsSold(Integer noTicketsSold) {
        this.noTicketsSold = noTicketsSold;
    }

    @Override
    public String toString() {
        return "Show{" +
                "showName='" + showName + '\'' +
                ", artistName='" + artistName + '\'' +
                ", date=" + date +
                ", placeOfShow='" + placeOfShow + '\'' +
                ", noTicketsAvailable=" + noTicketsAvailable +
                ", noTicketsSold=" + noTicketsSold +
                '}';
    }
}

File: an2/Medii de proiectare si programare/C#/tema7/model/domains/Show.cs
using System;

namespace model.domains
{
    [Serializable]
    public class Show : Entity<string>
    {
        public Show(string showName, string artist, DateTime date, string placeOfShow, int noTicketsAvailable,
            int noTicketsSold)
        {
            ShowName = showName;
            Artist = artist;
            Date = date;
            PlaceOfShow = placeOfShow;
            NoTicketsAvailable = noTicketsAvailable;
            NoTicketsSold = noTicketsSold;
        }

        public string ShowName { get; set; }
        public string Artist { get; set; }

        public DateTime Date { get; set; }
        public string PlaceOfShow { get; set; }
        public int NoTicketsAvailable { get; set; }
        public int NoTicketsSold { get; set; }

        public override string ToString()
        {
            return ShowName;
        }
    }
}